shreya_custom_loss_test.py

```diff
@@ -0,0 +1,73 @@
+from os import remove
+from tkinter.font import names
+
+# from numpy import error_message
+
+from ansys.motorcad.core import MotorCAD
+
+mc = MotorCAD(open_new_instance=False)
+
+# mc.do_magnetic_calculation()
+
+def add_internal_custom_loss(name, function, type, thermal_node):
+    if type not in ["Electrical", "Mechanical"]:
+        raise ValueError("Thermal Loss Type must be Electrical or Mechanical")
+    else:
+        # Find number of losses
+        no_internal_losses = mc.get_variable("NumCustomLossesInternal_Lab")
+        # Increase array length by 1
+        mc.set_variable("NumCustomLossesInternal_Lab", no_internal_losses + 1)
+        # Define properties
+        mc.set_array_variable("CustomLoss_name_internal_lab", no_internal_losses, name)
```

Comment:
When we have strings we are repeatedly using I would suggest making a global constant that we can use instead

```python
kCustomLoss_name_internal_lab = "CustomLoss_name_internal_lab"
```